clang/test/Driver/config-file.c

```diff
@@ -82,3 +82,35 @@
 // CHECK-TWO-CONFIGS: -isysroot
 // CHECK-TWO-CONFIGS-SAME: /opt/data
 // CHECK-TWO-CONFIGS-SAME: -Wall
+
+//--- The linker input flags should be moved to the end of input list and appear only when linking.
+// RUN: %clang --target=aarch64-unknown-linux-gnu --config %S/Inputs/config-l.cfg -o %s.out %s -### 2>&1 | FileCheck %s -check-prefix CHECK-LINKING
```

Comment:
`-o %s.out` is strange. We typically use `-o %t` for output. That said, since we use -### and the output isn't useful, just omit it.

Reply:
Add something like `-lxxx` to test the position of config specified `-l`.

Reply:
Thanks for all the comments.
Added both things.